flow/record/adapter/elastic.py

```diff
@@ -149,11 +149,14 @@ def streaming_bulk_thread(self) -> None:
             raise_on_exception=False,
```

Comment:
I wonder if we cannot just set `raise_on_error=True` and  `raise_on_exception=True` in `streaming_bulk`, would that also work?